editor/src/clj/editor/properties.clj

```diff
@@ -626,7 +626,9 @@
       v0)))
 
 (defn overridden? [property]
-  (and (contains? property :original-values) (not-every? nil? (:values property))))
+  (and (contains? property :original-values)
+       (every? some? (:original-values property))
+       (not-every? nil? (:values property))))
```

Comment:
This would now return false if some of the properties are not overridable. That shouldn't break anything as far as I understand, since the only place we use this pred is on `preperties_view`, to decide if the reset button is going to be displayed.

We could also modify `clear-override!` to always show the button if there is an overridden prop, and filter out the rest to avoid the exception.

Reply:
Makes sense!